trollflow2/plugins/__init__.py

```diff
@@ -123,6 +123,10 @@ def resample(job):
                 job['resampled_scenes'][area] = scn.resample(scn.max_area(),
                                                              **area_conf)
             else:
+                # The composites need to be created for the saving to work
+                if not set(scn.datasets.keys()).issuperset(scn.wishlist):
+                    LOG.debug("Generating composites for 'null' area.")
+                    scn.load(scn.wishlist, generate=True)
```

Comment:
You are not passing `area_conf`, could this be a problem ? Maybe just pass it with a modified `generate` value ?

Reply:
Are there parameters in `area_conf` that would affect loading for the full disk (or swath) area? In `load_composites()` the only additional kwarg passed to `scn.load()` is `resolution`, so maybe that is needed?

Reply:
Yes, and other read-specific keyword, eg masking space-pixels for H8...

Reply:
So those other read specific kwargs are also missing from the `load_composites()` call to `scn.load()`? I'll add the resolution kwarg here, the others need another PR (not sure what are really needed in which case).

Reply:
ah, sorry, I see now that `area_conf` is passed to the resampling

Reply:
so we probably just need to pickup the parameters from `load_composites` here instead.

Reply:
Oh, but all the relevant stuff (resolution etc.) are already present in `scn.wishlist` which contains a list of `DatasetID`s.

Reply:
ok then. For reference, that's what is done in `load_composites`: 
```python
scn.load(composites, resolution=resolution, generate=generate)
```